website/docs/components/app-header/partials/code/how-to-use.md

```diff
@@ -0,0 +1,167 @@
+## How to use this component
+
+The `AppHeader` provides persistent global navigation controls and utility links such as to help and user menus. It is meant to be paired with the `SideNav` which provides local or page-level navigation.
+
+!!! Insight
+
+The `AppHeader` is intended to be used in combination with the [`Hds::AppFrame`](/layouts/app-frame) component within the `Frame.Header` contextual component.
+
+!!!
+
+### Layout
+
+The AppHeader exposes three “slots” (named blocks) where consumers can yield the navigation content and add business logic to control the content.
+
+The `<:globalItemsBefore>` block should contain the `AppHeader::HomeLink` which is provided as a child component.
+
+The other two slots are used for consumer provided controls. The `<:globalItemsAfter>` block typically should provide an “OrgSwitcher” or “ProjectSwitcher“ control while the `<:>utilityItems>` block should contain utilities such “Help” and “User” menus and optionally “Search”.
```

Comment:
Small typo here I think for the named block.

```suggestion
The other two slots are used for consumer provided controls. The `<:globalItemsAfter>` block typically should provide an “OrgSwitcher” or “ProjectSwitcher“ control while the `<:utilityItems>` block should contain utilities such “Help” and “User” menus and optionally “Search”.
```

Reply:
Thanks, I fixed it along with other updates I pushed